Resources/ProxyProvider/DIRACCAProxyProvider.py

```diff
@@ -180,7 +184,7 @@ def getUserDN(self, userDict=None, sessionDict=None, userDN=None):
       return result
     caDN = result['Value']['subject']
     caDict = dict([field.split('=') for field in caDN.lstrip('/').split('/')])
-    caFieldByNid = dict([[self.fs2nid[field], field] for field in caDict])
+    self.caFieldByNid = dict([[self.fs2nid[field], field] for field in caDict])
```

Comment:
Why is this becoming a member variable? If it's for another PR it should be done in that PR
```suggestion
    caFieldByNid = dict([[self.fs2nid[field], field] for field in caDict])
```

Reply:
yes, I missed it. I'l fix it.